src/OrchardCore.Modules/OrchardCore.Resources/wwwroot/Styles/bootstrap.css

```diff
@@ -4,10 +4,10 @@
 */
 
 /*!
- * Bootstrap v4.5.2 (https://getbootstrap.com/)
+ * Bootstrap v4.5.0 (https://getbootstrap.com/)
```

Comment:
This looks like a regression here

Reply:
Ping!

Reply:
You might need to merge and rebuild

Reply:
This comment is on an outdated file.

The PR file has the correct version v4.5.2 merged.